CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+### 1.11.1
```

Comment:
I think we should have the same message as `1.11.0` about automated list of changes